Modules/rat_rescue.py

```diff
@@ -502,23 +529,20 @@ def unidentified_rats(self, value) -> None:
             raise TypeError(f"expected type str, got {type(value)}")
 
     @property
-    def is_open(self) -> bool:
+    def open(self) -> bool:
         """
-        Bool storing the Rescue's open status.
-
-        - this cannot be named `Rescue.open` as it would shadow the name from
-        the outer scope (bad)
+        Helper method for determining if a case is considered open or not
 
         Returns:
             bool: is case open?
 
         """
-        return self._open
+        return False if self.status is Status.CLOSED else True
```

Comment:
`return self.status is not Status.CLOSED`?

Reply:
`Status` are not booleans.

Reply:
No shit Sherlock. 
In glorious Python fashion, `is not` is a single operator and returns the opposite of `is` in all cases. Including testing two references against each other.

Reply:
Here, it would be equivalent to using `!=`

Reply:
oh wait, i misread you completely... :/
yes `is not` should be the implementation. will fix